### ISSUE TITLE
FakeSMC Kernel Panic with v0.5.9

### ISSUE THREAD

**Issue by twlawrenceko** [acidanthera/bugtracker#953]
Hello,

I know this may not be highly supported,  however, due to less reading of sensor values for Dell laptops,  I may use FakeSMC as a short time replacement of VirtualSMC. 

As I use FakeSMC (removed VirtualSMC), it results in KP. Is this due to VirtualSMC.efi built in inside v0.5.9?

Any help is highly appreciated. 

Twsps

**Comment by Andrey1970AppleLife** [comment]
Now I will enter a trance and through space I will penetrate into your computer to see your panic log.

**Comment by twlawrenceko** [comment]
Here is the log. It shows halting on critical error.

[opencore-2020-06-05-000656.txt](https://github.com/acidanthera/bugtracker/files/4733240/opencore-2020-06-05-000656.txt)

**Comment by vit9696** [comment]
89:728 00:173 OCAK: Plist-only kext has CFBundleExecutable key

Mistake in kext injection in config.

**Comment by twlawrenceko** [comment]
> 89:728 00:173 OCAK: Plist-only kext has CFBundleExecutable key
> 
> Mistake in kext injection in config.

It is halting due to FakeSMC_ACPISensor.kext. I have moved it right after FakeSMC.kext but still results in the same error.

**Comment by twlawrenceko** [comment]
> 89:728 00:173 OCAK: Plist-only kext has CFBundleExecutable key
> 
> Mistake in kext injection in config.

It was solved by using "https://github.com/CloverHackyColor/FakeSMC3_with_plugins"

I also have an issue where boot chime shows invalid parameter. I have followed the guide but still no sound. Here is a new boot log. 

`35:502 00:085 OCAU: Language for audio en-US:0 - Success
35:592 00:090 OC: Starting to play chime...
35:679 00:087 OC: Wave Resources\Audio\OCEFIAudio_VoiceOver_Boot.wav was requested
35:888 00:209 OCAU: File 59 for lang 7 is 32 2 2 (440576) - Success
35:979 00:090 OCAU: PlayFile playback setup failure - Invalid Parameter
35:979 00:000 OC: Play chime started - Invalid Parameter`
[opencore-2020-06-05-022106.txt](https://github.com/acidanthera/bugtracker/files/4733589/opencore-2020-06-05-022106.txt)
